static/app/components/overlay.tsx

```diff
@@ -108,15 +109,16 @@ const Overlay = styled(
       },
       ref
     ) => {
-      const animationProps = animated
-        ? {
-            ...overlayAnimation,
-            style: {
-              ...style,
-              ...computeOriginFromArrow(placement, originPoint),
-            },
-          }
-        : {style};
+      const animationProps =
+        !IS_ACCEPTANCE_TEST && NODE_ENV !== 'test' && animated
```

Comment:
This is gnarly

I see these props are heading into `<motion.div>`. Is there a way to   mock that component out, or some way to make it ignore the non-static props when it's under test? That way components won't need to know about these tests consts.

Reply:
I agree, but at the same time it keeps the disable the same for both acceptance tests and jest tests? I pulled it into a variable is that any better

Reply:
ah acceptance tests would be a different runner by similar ideas :(
i didn't think about that.